tiledb/sm/enums/query_status.h

```diff
@@ -44,6 +49,44 @@ enum class QueryStatus : char {
 #undef TILEDB_QUERY_STATUS_ENUM
 };
 
+/** Returns the string representation of the input querystatus type. */
+inline const std::string& query_Status_str(QueryStatus query_Status) {
```

Comment:
Looks like copy/paste error with the case change (capital `S`).

Reply:
Good catch, fixed.